examples/patterns/navigation/light.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Navigation / Fixed width
+category: _patterns
+---
+
+<header id="navigation" class="p-navigation--light" role="banner">
+  <div class="row">
+    <div class="p-navigation__logo">
+      <a class="p-navigation__link" href="#">
+        Vanilla
+      </a>
+    </div>
+    <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+    <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Menu</a>
+    <nav class="p-navigation__nav">
```

Comment:
We should be looking to improve the a11y for the navigation, we fail accessiblity if we use two toggle links. I suggest we change these to ```button``` elements and use aria tags to signify whether it's expanded or not.